Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
seems like CF_INLINE still works just fine!

---
In reply to: [78862337](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r78862337) [](ancestors = 78862337)